cron/poll_read_issue_comments.py

```diff
@@ -36,16 +36,31 @@ def get_seconds_remaining(api, comment_id):
     return seconds_remaining
 
 
+def insert_or_update_issue(api, issue_id, number):
+    # get more info on the issue
+    gh_issue = gh.issue.open_issue(api, settings.URN, number)
+
+    # get user from db
+    user, _ = User.get_or_create(user_id=gh_issue["user"]["id"],
+                                 defaults={"login": gh_issue["user"]["login"]})
+
+    # db insert
+    issue, _ = Issue.get_or_create(issue_id=issue_id,
+                                   number=number,
```

Comment:
I would recommend using the `defaults` param here for anything not in the primary key. We were having issues using this syntax before